fipy/viewers/matplotlibViewer/matplotlib2DGridContourViewer.py

```diff
@@ -84,25 +101,28 @@ def _plot(self):
         Z = self.vars[0].value
         X, Y, Z = [v.reshape(shape, order='F') for v in (X, Y, Z)]
 
-        numberOfContours = 10
-        smallNumber = 1e-7
-        zmin = self.norm.vmin
-        zmax = self.norm.vmax
-        diff = zmax - zmin
-
-        if diff < smallNumber:
-            V = numerix.arange(numberOfContours + 1) * smallNumber / numberOfContours + zmin
-        else:
-            V = numerix.arange(numberOfContours + 1) * diff / numberOfContours + zmin
```

Comment:
Is this logic being removed as matplotlib takes care of it now?

Reply:
Yes. I don't know if it always did, but it certainly does a better job than we do now.